### ISSUE TITLE
Come up with standard structure for pulling in tutorials

### ISSUE THREAD

**Issue by patrickmineault** [NeuromatchAcademy/course-content#58]
The naming scheme we have right now in tutorials won't work for a high number of tutorials. Instead of placing the Bayes day tutorials in the folder called Bayes, how about:

WxDy-SnakeCaseSubject, .e.g. W2D1-BayesianStatistics

Inside, we can have the student version at the top level, the solution inside of TA_solutions, and static content specific for that day inside of static.

**Comment by mpbrigham** [comment]
Yup, makes sense. How do we go about this @patrickmineault? Update the `writing-tutorials.md` page 
 to detail these instructions for content branches?

**Comment by mwaskom** [comment]
Porting discussion from #61 

I am :+1: on the naming scheme, but treating the student version of the tutorial as canonical is going to get us into problems when we want to iterate on the notebooks.

I think we can avoid conflicts with ongoing development if we invert the organization of the directory and treat the full version of the notebook as the canonical version and the student version (with removed solutions) as the derivative.

So the directory structure would look like

```
W{n}D{m}-{topic}/
    student/
    static/
    W{n}D{m}_Tutorial{i}_{name}.ipynb
```

I'm not sure if putting the student versions in a subdirectory will make it annoying to get the notebooks loaded up for everyone to use in colab.

Another option would be flat:

```
W{n}D{m}-{topic}/
    complete/
    student/
    static/
```

**Comment by patrickmineault** [comment]
The first structure in your last post makes sense to me - I don't think it's going to make it more annoying, because we will have badges.

**Comment by mwaskom** [comment]
One advantage to the second ("flat") organization is that we could have the notebook processing script write out the post-execution version of the notebook. That would ensure that execution numbers are in order and mean that the complete/TA version matches the student version more closely. In principle this shouldn't be a problem if the content creators are sending freshly-executed notebooks into github, but automation might be our friend here.

**Comment by mwaskom** [comment]
I could also just have the CI fail if it gets a notebook with non-sequential execution ids, maybe that's the easiest way to enforce that the inputs look like the outputs.

**Comment by mwaskom** [comment]
Hah it looks like colab *does* do a force push when it "saves a copy to github".

**Comment by mwaskom** [comment]
Closed with #63
